src/main/scala/cromwell/binding/package.scala

```diff
@@ -13,5 +13,7 @@ import cromwell.binding.values.WdlValue
  */
 
 package object binding {
+  type Wdl = String
```

Comment:
To paraphrase me earlier - a meteor isn't going to strike the Earth if it's not there. OTOH we're able to type alias for a single LOC and it's slightly more expressive